services/order.py

```diff
@@ -0,0 +1,45 @@
+from django.core.exceptions import ValidationError
+from django.db import transaction
+from django.utils.datetime_safe import datetime
+
+from db.models import Order, User, Ticket
+
+
+def create_order(tickets: list, username: str, date: datetime = None) -> Order:
+    if not tickets:
+        raise Exception("No tickets to create")
+
+    with transaction.atomic():
+        user = User.objects.get(username=username)

```

Comment:
The `User.objects.get(username=username)` call can raise a `DoesNotExist` exception if the user is not found. Consider handling this exception to provide a more user-friendly error message.